llama-index-core/tests/workflow/test_streaming.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+import pytest
+
+from llama_index.llms.openai import OpenAI
+
+from llama_index.core.llms import ChatMessage, MessageRole
+from llama_index.core.workflow.context import Context
+from llama_index.core.workflow.decorators import step
+from llama_index.core.workflow.events import Event, StartEvent, StopEvent
+from llama_index.core.workflow.workflow import Workflow
+from llama_index.core.workflow.errors import WorkflowRuntimeError
+
+
```

Comment:
should we include a `@pytest.mark.skipif(...)` if `openai` is not installed?

[example](https://github.com/run-llama/llama_index/blob/31ce06c70561ab74066ed45dfa7ad0cf16f8e589/llama-index-core/tests/memory/test_chat_summary_memory_buffer.py#L126)

Reply:
Actually we don't need OpenAI at all :)